rules/windows/defense_evasion_root_dir_ads_creation.toml

```
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2024/03/14"
+integration = ["endpoint", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of an Alternate Data Stream (ADS) at the volume root directory (C:), which can indicate the
+attempt to hide tools and malware, as ADSs created in this directory are not displayed by system utilities.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.sysmon_operational-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Alternate Data Stream Creation at Volume Root Directory"
+references = ["https://www.crowdstrike.com/blog/anatomy-of-alpha-spider-ransomware/"]
+risk_score = 47
+rule_id = "ff6cf8b9-b76c-4cc1-ac1b-4935164d1029"
+severity = "medium"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend", "Data Source: Sysmon"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where host.os.type == "windows" and event.type == "creation" and
+  startsWith(file.path, "C:\\:")
```

to catch both file or process execution, `regex~` is already case insensitive.


```suggestion
any where host.os.type == "windows" and event.category in ("file", "process") and 
 (
   (event.type == "creation" and file.path regex~ """[A-Z]:\\:.+""") or 
   
   (event.type == "start" and process.executable regex~ """[A-Z]:\\:.+""")
   )
```